config/local_defaults.go

```diff
@@ -30,24 +30,17 @@ const configVersion = uint32(4)
 // These versioned structures need to be maintained CAREFULLY and treated
 // like UNIVERSAL CONSTANTS - they should not be modified once committed.
 //
-// Add fields or change defaults in a new defaultLocalV# instance,
-// bump the version number (configVersion), and add appropriate migration and tests.
-//
-// EXCEPTION: If you are adding a new value whose default value is it's implicit default value
-//            it is safe / acceptable to add it without bumping the Config version.
-//
-// CLARIFICATION:
+// New fields may be added to the current defaultLocalV#. If they have
+// a value other than the type default zero value add them to the
```

Comment:
Migration not required.

Reply:
Someone could have a V(N-1) config file in place and I think they would need `migrate()` to get an actual value for a non-zero config in V(N)

Reply:
So after several iterations of getting this to work as desired, that is not true.

The current consumption of config.json looks like this:

- Start with LatestDefaults
- Reset Version to 0
- Load config.json (which overwrites explicitly-defined values)
- Process migrations

This does exactly what everyone assumed would happen when config.json was modified in the past.